Portals/_default/Skins/Vanjaro/Base.ascx.cs

```diff
@@ -124,6 +124,10 @@ protected void Page_Load(object sender, EventArgs e)
         {
             JavaScript.RequestRegistration(CommonJs.jQuery, null, SpecificVersion.Latest);
 
+            string ThemeJS = "~/Portals/_default/vThemes/" + Core.Managers.ThemeManager.CurrentTheme.Name + "/theme.js";
+            if (File.Exists(System.Web.Hosting.HostingEnvironment.MapPath(ThemeJS)))
```

Comment:
I'm not sure if we need the File.Exists check. I believe that's done internally in ClientResourceManager. Please confirm by removing the JS file and registering it with ClientResourceManager; there should not be any 404 in console.

Reply:
![image](https://user-images.githubusercontent.com/62750280/95545478-98ddce80-0a1b-11eb-9acd-6483e0ff241e.png)
I can confirm it is throwing 404 error.

Reply:
Okay. Let's refactor the ThemeManager.CurrentTheme to include ThemeManager.CurrenTheme.HasScript() to return true or false by doing a File.Exists check. It should also cache the result so it's only checked once.